utils.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	ifaceNameTooLongErr       = fmt.Errorf("Interface name length must be between 1 and 16 characters long")
+	ifaceNameContainsSpaceErr = fmt.Errorf("Interface name cannot contain spaces")
+	ifaceNameContainsSlashErr = fmt.Errorf("Interface name cannot contain '/' character")
+)
+
+func verifyInterfaceName(name string) error {
+	if len(name) == 0 || len(name) > 16 {
```

Comment:
We need to subtract the length of the prefix from this, so actually its 6.

Reply:
Also - would be nice to add a comment where this logic came from, so either https://unix.stackexchange.com/questions/451368/allowed-chars-in-linux-network-interface-names or https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/tree/lib/utils.c?id=1f420318bda3cc62156e89e1b56d60cc744b48ad#n827 ?